base/src/test/kotlin/io/spine/util/TextTest.kt

```diff
@@ -75,4 +75,12 @@ internal class TextTest {
         assertThrows<IllegalArgumentException> {  Text.of("un", "${nl}o") }
         assertThrows<IllegalArgumentException> {  Text(listOf("dos", "tres${nl}")) }
     }
+
+    @Test
+    fun `always return the same value`() {
+        val text = Text.of("donna", "be", "la", "mare")
+
+        val value = text.value()
+        assertThat(value).isSameInstanceAs(text.value())
```

Comment:
I think this test will pass even if we return a new instance every time, since `String` itself has a pool of re-usable instances.

Reply:
Then let's keep it just a requirement. Don't want to involve mocking just for this.